frontend/src/elm/Main/Models.elm

```diff
@@ -0,0 +1,120 @@
+module Main.Models exposing (..)
+
+
+type alias Model =
+  { majors : List Major
+  , state : Int
+  }
+
+type alias Major =
```

Comment:
I think the name of type `Major` is not proper.
This type means the result of simulation based on a major, not a major itself.